### ISSUE TITLE
Putting arrays into tenant storage is problematic

### ISSUE THREAD

**Issue by stancl** [archtechx/tenancy#26]
```php
>>> tenancy()->put('foo', [1, 2]);
=> [
     1,
     2,
   ]
>>> tenancy()->get('foo')
=> "Array"
```

Arrays should be serialized into JSON, but then JSON strings need to be recognized from regular strings.

**Comment by stancl** [comment]
```php
>>> json_encode("abc")
=> ""abc""
>>> json_encode(['a' => 'b', 'c' => 'd'], true)
=> "{"a":"b","c":"d"}"
>>> json_encode(7)
=> "7"
>>> json_decode(json_encode(7))
=> 7
```

I'll just `json_encode()` all data.

**Comment by stancl** [comment]
Fixed in #27.
